resources/Materials/TestSuite/stdlib/texture/image.mtlx

```diff
@@ -41,10 +41,4 @@ Basic image unit test with one image node for each variation in input type.
   </image>
   <output name="image_float_output" type="float" nodename="image_float" />
 
-   <image name="image_default" type="color3">
```

Comment:
MDL will error on during compilation as image file is not found, so remove test.